controllers/providers/aws/aws.go

```diff
@@ -201,6 +201,44 @@ func (w *AwsWorker) DeleteScalingGroup(name string) error {
 	return nil
 }
 
+func (w *AwsWorker) SetSuspendProcesses(name string, processesToSuspend []string) error {
+
+	var scalingProcesses []*string
+
+	for _, process := range processesToSuspend {
+		scalingProcesses = append(scalingProcesses, aws.String(process))
+	}
+
+	input := &autoscaling.ScalingProcessQuery{
+		AutoScalingGroupName: aws.String(name),
+		ScalingProcesses:     scalingProcesses,
+	}
+	_, err := w.AsgClient.SuspendProcesses(input)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func (w *AwsWorker) SetResumeProcesses(name string, processesToResume []string) error {
+
+	var scalingProcesses []*string
+
+	for _, process := range processesToResume {
+		scalingProcesses = append(scalingProcesses, aws.String(process))
+	}
+
+	input := &autoscaling.ScalingProcessQuery{
+		AutoScalingGroupName: aws.String(name),
+		ScalingProcesses:     scalingProcesses,
```

Comment:
You can probably just use `aws.StringSlice(processesToResume)` and remove the iteration in the top